src/aircraft_cmd.cpp

```diff
@@ -1329,6 +1329,7 @@ static void CrashAirplane(Aircraft *v)
 	const Station *st = GetTargetAirportIfValid(v);
 	StringID newsitem;
 	TileIndex vt;
+	NewsType newstype = NT_ACCIDENT;
```

Comment:
Move this variable declaration slightly closer to where it is used?

Reply:
@LordAro updated, thanks